generator.json

```diff
@@ -1797,6 +1797,94 @@
                 "$nuget/Microsoft.Direct3D.DirectStorage/*/$version": "src/Native/Silk.NET.DirectStorage.Native/version.txt"
             }
         },
+        {
+            "profileName": "DirectComposition",
+            "sources": [
+				"build/include/silk.net.directcomposition.dcomp.h"
+            ],
+            "mode": "Clang",
+            "cacheDir": "build/cache",
+            "cacheKey": "dcomp",
+            "controlDescriptors": [
+                "convert-windows-only"
```

Comment:
Can you add the typemap native and no obsete enum control descriptors? the first has been retroactively applied on most Direct* ones and the second is applied to new ones, if typemap-native causes problems though, it can be ignored, aside from that it looks good